### PR TITLE
Created AllActivitiesScreen and MapScreen

### PR DESCRIPTION
**Pull request by varnitg**
Created AllActivitiesScreen and MapScreen

### CODE REVIEW

File: client/app/screens/MapScreen.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import {
+	View,
+	Text,
+} from 'react-native';
+
+
+class MapScreen extends React.Component {
```
Comment (AbhiPrasad) [inline client/app/screens/MapScreen.js]: Did you delete the MapScreen component from the Container file?

File: client/app/screens/MapScreen.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	/* eslint-enable */
```
Comment (AbhiPrasad) [inline client/app/screens/MapScreen.js]: You can probably remove all the `eslint-disable` and `eslint-enable` now.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (AbhiPrasad) [inline package-lock.json]: Remove this file!

**Comment by amyqiu** [comment]
Looks like there are some conflicts that you might want to fix? And you should take out the package-lock.json file

**Comment by theRoughCode** [comment]
Updated PR at #84

**Pull request closed without merging**